stubs/ttkthemes/ttkthemes/_utils.pyi

```diff
@@ -1,7 +1,7 @@
-from _typeshed import Incomplete
+from _typeshed import FileDescriptorOrPath, StrOrBytesPath
 
-def temporary_chdir(new_dir) -> None: ...
-def get_file_directory(): ...
-def get_temp_directory(): ...
-def get_themes_directory(theme_name: Incomplete | None = None, png: bool = False): ...
-def create_directory(directory): ...
+def temporary_chdir(new_dir: FileDescriptorOrPath) -> None: ...
```

Comment:
This actually returns a `ContextManager`, since it is decorated with `@contextmanager`.

That said, I think we should delete `_imgops.pyi` and `_utils.pyi` from typeshed. They are not intended to be imported by users of this library.

Reply:
choosing between `AbstractContextManager[None]` and `Generator[None]` I took the latter (P.S.: `ContextManager` from `typing` is deprecated)

I don't think the types for these functions will get in the way for our type stubs, and many 3rd party libs have types for utils functions

I suggest leaving them

Reply:
`Generator[None]` is wrong because that's not what the function returns when called.

Reply:
sorry, changed to `AbstractContextManager[None]`.
 it was a suggestion from my lsp server :)